server/src/main/java/io/unitycatalog/server/UnityCatalogServer.java

```diff
@@ -100,6 +108,7 @@ private void addServices(ServerBuilder sb) {
         new TemporaryModelVersionCredentialsService(credentialOperations);
     sb.service("/", (ctx, req) -> HttpResponse.of("Hello, Unity Catalog!"))
         .annotatedService(controlPath + "auth", authService, unityConverterFunction)
+        .annotatedService(controlPath + "scim2/Users", Scim2UserService)
```

Comment:
Without the `unityConverterFunction`, doesn't the SCIM API fail on unknown properties? Is that what we want?

Reply:
We should probably make it do the same thing. Actually there's some other special serialization for SCIM, so I'll probably make a different converter function.